qiskit_experiments/database_service/db_fitval.py

```diff
@@ -23,6 +23,8 @@ class FitVal:
     This data is serializable with the Qiskit Experiment json serializer.
     """
 
+    # TODO deprecate this (replace with UFloat)
```

Comment:
If we are going to leave TODOs in the code we should make them more explicit. Can you state what needs to be done here. Perhaps even in an issue and reference it?

Reply:
Done in https://github.com/Qiskit/qiskit-experiments/pull/551/commits/6e149e99ad20c5e2e2f51683b92383d8dbe010d2